moveit_experimental/moveit_jog_arm/test/arm_setup/config/jog_settings.yaml

```diff
@@ -1,33 +1,55 @@
-incoming_command_timeout:  1  # Stop jogging if X seconds elapsed without a new cmd
-joint_topic:  joint_states
-move_group_name:  panda_arm  # Often 'manipulator' or 'arm'
-lower_singularity_threshold:  30  # Start decelerating when the condition number hits this (close to singularity). Larger --> closer to singularity
-hard_stop_singularity_threshold: 45 # Stop when the condition number hits this. Larger --> closer to singularity
-collision_proximity_threshold: 0.03 # Start decelerating when a collision is this far [m]
-planning_frame: base_link  # The MoveIt planning frame. Often 'base_link'
-low_pass_filter_coeff: 2.  # Larger-> more smoothing to jog commands, but more lag.
+scale:
+  linear:  0.003  # Max linear velocity. Meters per publish_period. Units is [m/s]. Only used if command_in_type=="unitless"
+  rotational:  0.006 # Max angular velocity. Rads per publish_period. Units is [rad/s]. Only used if command_in_type=="unitless"
+  joint: 0.01  # Max joint angular/linear velocity. Rads or Meters per publish period. Units is [rad/s] or [m/s].
+low_pass_filter_coeff: 2.  # Larger --> trust the filtered data more, trust the measurements less.
+
+## Properties of outgoing commands
 publish_period: 0.01  # 1/Nominal publish rate [seconds]
-collision_check_rate: 5 # [Hz] Collision-checking can easily bog down a CPU if done too often.
-num_halt_msgs_to_publish: 1 # If 0, republish commands forever even if the robot is stationary. Otherwise, specify num. to publish
-# Publish boolean warnings to this topic
-warning_topic: jog_server/warning
-joint_limit_margin: 0.1 # added as a buffer to joint limits [radians]. If moving quickly, make this larger.
-command_out_topic: jog_server/command # Find this topic by 'rostopic list' or 'rostopic list | grep command' or 'rqt_graph'
+publish_delay: 0.005  # delay between calculation and execution start of command
+
 # What type of topic does your robot driver expect?
-# Currently supported are std_msgs/Float64MultiArray (for ros_control JointGroupVelocityController)
+# Currently supported are std_msgs/Float64MultiArray (for ros_control JointGroupVelocityController or JointGroupPositionController)
 # or trajectory_msgs/JointTrajectory (for Universal Robots and other non-ros_control robots)
 command_out_type: trajectory_msgs/JointTrajectory
-# Can save some bandwidth as most robots only require positions or velocities
+
+# What to publish? Can save some bandwidth as most robots only require positions or velocities
 publish_joint_positions: true
 publish_joint_velocities: true
 publish_joint_accelerations: false
+
+## MoveIt properties
+move_group_name:  panda_arm  # Often 'manipulator' or 'arm'
+planning_frame: base_link  # The MoveIt planning frame. Often 'base_link'
+
+## Stopping behaviour
+incoming_command_timeout:  1  # Stop jogging if X seconds elapse without a new cmd
+# If 0, republish commands forever even if the robot is stationary. Otherwise, specify num. to publish.
```

Comment:
OK, I'll spell out command.

I think you are rightly confused because `incoming_command_timeout` applies to incoming commands, `num_halt_msgs_to_publish` applies to outgoing commands. So they don't affect each other.

Reply:
I'll rename `num_halt_msgs_to_publish` -> `num_outgoing_halt_msgs_to_publish`